src/backup-metadata/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Build the backup-metadata binary
```

Comment:
Should switch over to using buildpacks/pack to build images instead of Dockerfiles (consistent with how we're building pretty much all of the CAPI team's images)

Reply:
yup. I am not sure how to do this, so would be something we should do during an integration call / pairing session I think. Or if it's easy to do, some guidance would be appreciated 👍